tests/operator_test.go

```diff
@@ -1003,11 +1003,31 @@ var _ = Describe("ALL Operator tests", func() {
 						for _, rule := range group.Rules {
 							if len(rule.Alert) > 0 {
 								Expect(rule.Annotations).ToNot(BeNil())
-								url, ok := rule.Annotations["runbook_url"]
-								Expect(ok).To(BeTrue())
-								resp, err := http.Head(url)
-								Expect(err).ToNot(HaveOccurred())
-								Expect(resp.StatusCode).Should(Equal(http.StatusOK))
+								checkForRunbookURL(rule)
+								checkForSummary(rule)
+							}
+						}
+					}
+				}
+			})
+
+			It("[test_id:8812] Alerts should have all the requried labels", func() {
+				promRule := &promv1.PrometheusRule{
+					ObjectMeta: metav1.ObjectMeta{
+						Name:      "prometheus-cdi-rules",
+						Namespace: f.CdiInstallNs,
+					},
+				}
+				err := f.CrClient.Get(context.TODO(), crclient.ObjectKeyFromObject(promRule), promRule)
+				Expect(err).ToNot(HaveOccurred())
+				for _, group := range promRule.Spec.Groups {
```

Comment:
Noticed we have a bunch of for loops in alert tests, maybe we should ensure they match our expectations?

Thanks to @machadovilaca, 
I believe the alerting rules are gathered in one place for us to grab and `==` on their len:
`getAlertRules()`
We can have this Expect outside the loop so we know we definitely had all alerts present

Reply:
As agreed, will be implemented in a different PR.